objects/data/AlphaVantage.js

```diff
@@ -41,6 +41,7 @@ class AlphaVantage {
 				else if (response.statusCode !== 200) reject(body);
 				else {
 					const json = JSON.parse(body);
+					if(typeof json.Note === "string") if(json.Note.substring(0,45) == "Thank you for using Alpha Vantage! Our standa") reject(new LibraryError("max requests"));
```

Comment:
1 if per line, please. And for any nesting please use curly braces.

Feel free to use an `&&`.

Space after if before parentheses.